{"name": "test_fail_2", "status": "skipped", "statusDetails": {"message": "XFAIL\n\nAssertionError: assert False", "trace": "@pytest.mark.xfail\n    def test_fail_2():\n>       assert False\nE       assert False\n\nPytestSamples\\test_AllureReports.py:26: AssertionError"}, "start": 1662321414929, "stop": 1662321414929, "uuid": "faf9503a-4367-4022-b363-70b14d766eff", "historyId": "27515ee70c2e51893000b4df90df811d", "testCaseId": "72d69330fdde61215b08863a3a7ac7ec", "fullName": "PytestSamples.test_AllureReports#test_fail_2", "labels": [{"name": "tag", "value": "@pytest.mark.xfail"}, {"name": "parentSuite", "value": "PytestSamples"}, {"name": "suite", "value": "test_AllureReports"}, {"name": "host", "value": "DESKTOP-U63UI4D"}, {"name": "thread", "value": "12768-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "PytestSamples.test_AllureReports"}]}